aries-site/src/pages/components/status-indicator.mdx

```diff
@@ -0,0 +1,54 @@
+import { StatusGoodSmall } from 'grommet-icons/icons/StatusGoodSmall';
+import { StatusWarningSmall } from 'grommet-icons/icons/StatusWarningSmall';
+import { StatusCriticalSmall } from 'grommet-icons/icons/StatusCriticalSmall';
+import { StatusUnknownSmall } from 'grommet-icons/icons/StatusUnknownSmall';
+import { StatusBox } from '../../examples';
+
+import { Box } from 'grommet';
+
+## What makes up a status indicator?
+
+To effectively capture the user’s attention, status indicators are compromised of four different elements:
+
+- Colors
+- Icons
+- Shapes
+- Content
+
+To meet [WCAG 2.1 compliance](https://www.w3.org/WAI/WCAG21/quickref/#status-messages), at least three out of four elements must always be present.
+
+### Colors
+
+The colors used for Status are found within the [HPE Design System Color guidance](https://design-system.hpe.design/foundation/color#status-colors). Additionally, always pair color with other elements as it is not widely accessible by everyone.
+
+These are some typical situations in which each color could be used: 
+
+- A critical status used to alert a device failure
+- A warning status used to notify the user of an expiring license
+- A normal status used to communicate that an update has successfully completed
+- An unknown status used to indicate that a page cannot be found
+
+### Icons and Shapes
+
+Icons and shapes are used to represent the notification’s level of severity to users. Here are two sets of icons and shapes created for Statuses found within the [HPE Design System Icons](https://design-system.hpe.design/foundation/icons) page.
```

Comment:
Can we remove the comment around "two sets of icons" since it seems we're just using one set?